src/test/java/seedu/loyaltylift/model/customer/AddressTest.java

```diff
@@ -6,6 +6,8 @@
 
 import org.junit.jupiter.api.Test;
 
+import seedu.loyaltylift.model.attribute.Address;
```

Comment:
This test should be in `.../module/attribute` package instead of `.../module/customer`. This would also remove the need of this import.